docs/browser-support.rst

```diff
@@ -17,42 +22,105 @@ We seek to provide usable experiences of our most important web content to all u
     * Enhanced behavior is provided by unobtrusive, externally linked JavaScript
     * End-user web browser preferences are respected
 
-Some website experiences may require us to deviate from these principles -- imagine *a marketing campaign page built under timeline pressure to deliver novel functionality to a particular locale for a short while* -- but those will be exceptions and rare.
+Some website experiences may require us to deviate from these principles -- imagine *a
+marketing campaign page built under timeline pressure to deliver novel functionality to a
+particular locale for a short while* -- but those will be exceptions and rare.
 
-Technical details
------------------
+Browser Support Matrix (Updated 2019-05-11)
+-------------------------------------------
 
-We deliver enhanced CSS & JS to browsers in our browser support matrix (below). We deliver basic support to all other user agents.
-
-Basic support consists of no page-specific CSS or JS. Instead, we deliver basic semantic HTML, a universal CSS stylesheet that gets applied to all pages, and a universal JS bundle that only handles downloading Firefox (click a button, get a file), and Google Analytics.
-
-Browser Support Matrix (Updated 20190409)
------------------------------------------
+We deliver enhanced CSS & JS to browsers in our browser support matrix (below).
+We deliver basic support to all other user agents.
 
 **The following browsers have enhanced support:**
 
   * All evergreen browsers (Firefox, Chrome, Safari, Edge, Opera, etc.)
-  * IE10 and above.
+  * IE11 and above.
 
 **The following browsers have basic support:**
 
-  * All other IE browsers.
+  * Outdated evergreen browser versions.
```

Comment:
I think we need to add a 3rd class here. IE10 gets basic JS support but enhanced CSS support.

Reply:
I had hoped that definition of what basic support consists of is explained in enough detail below that we don’t necessarily need a 3rd class here. Even though IE10 is still served the same stylesheet, it doesn’t mean it gets enhanced CSS support by default. Using feature detection to deliver a degraded experience where needed is the most suitable method here.

I guess it depends if we think of basic support as a binary thing or not. It could mean “serving a universal stylesheet”, or it could mean “serving a degraded experience”.

I’m open to what others think makes the most sense.

Reply:
In my mind we have 3 types of browsers:

- those who get JS & CSS
- those which get CSS but not JS
- those that do not get CSS or JS

I think the most clear thing would be to be explicit about what each browser gets. But maybe I am thinking about this from a different angle?

Reply:
Or maybe we have two categories:

- does not get JS or CSS
- gets JS and CSS but only if they pass the feature tests.

In which case, IE 10 still belongs in the enhanced support catagory.

Reply:
> In my mind we have 3 types of browsers:

I think what we have is more in terms of:

- those who get 1st class JS & CSS.
- those who get a gracefully degraded experience.
- those who get a universal CSS / JS bundle.

I added a commit to update the docs that hopefully makes this a little clearer. Let me know what you think.